pegasus/sites.v3/code.org/public/markdown/advanced.moved

```diff
@@ -1 +0,0 @@
-/editing
```

Comment:
I think we need to keep the files in this directory, because of existing code references: https://github.com/code-dot-org/code-dot-org/blob/4449f68613ddcfcf79f864abad873bd8fc515ea6/pegasus/sites.v3/code.org/views/markdown_nav.haml#L4-L9

Reply:
Reinstated ✅